src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -175,7 +175,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                             }
                             else
                             {
-                                _logger.LogWarning(0, ex, $"Unable to bind to {parsedAddress.ToString()} on the IPv6 loopback interface.");
```

Comment:
`Unable to bind to http://localhost:27574 on the IPv6 loopback interface. (Error -4089 EAFNOSUPPORT address family not supported)`